ReSerialize.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/sh
-
-for f in ontology/*.ttl; do
-    echo "ReSerializing file: $f"
-    tools/ReSerialize/bin/ReSerialize -f $f -r
-    if [ $? -gt 0 ]; then
-        echo "> Modified file: $f"
-    fi
```

Comment:
Het lijkt me dat je dit nog wel wil, om te zien of er bestanden zijn aangepast (beetje feedback is dan fijn)